lms/djangoapps/email_marketing/tasks.py

```diff
@@ -454,6 +455,9 @@ def _build_purchase_item(course_id, course_url, cost_in_cents, mode, course_data
     if 'tags' in course_data:
         item['tags'] = course_data['tags']
 
+    # add vars to item
+    item['vars'] = dict(course_data.get('vars', {}), mode=mode, course_run_id=course_id._to_string())  # pylint: disable=W0212
```

Comment:
We should avoid the need to use `# pylint: disable=W0212`

Can you adjust the code to not need this?